resources/texts/_settingsManager.html

```diff
@@ -13,6 +13,10 @@ <h1>Settings</h1>
         <label>Port for local Kiwix server : </label>
```

Comment:
This string is not translated/translatable. All the strings should be properly translatable.

Reply:
I will fix this in another PR